logic/scripts/drops.py

```diff
@@ -0,0 +1,82 @@
+from contextlib import closing
+import json
+import webbrowser
+import re
+
+from bs4 import BeautifulSoup
+
+import requests
+from requests.exceptions import RequestException
+from tools import db_utils
+from util import tasks
+from config import constants
+from types import SimpleNamespace
+from datetime import date, datetime, timezone
+from dateutil import parser
+
+
+
+headers = {
```

Comment:
I'm curious. Are those headers actually required for the API call against the NFT server to succeed? Seems weird they would?

Reply:
I guess `application/json` is default so no... I removed them